### PR TITLE
Fetch event detail from sessions

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Publish/Program.fs [hunk 1]
```diff
@@ -19,21 +19,116 @@ module JsonHttpClient =
...
+              SpeakerRating = speaker.Rating
+              StartDate = session.Date
+              EndDate = session.Date |> Option.map (fun date -> date.AddHours(1.0)) }
+
+    module Event = 
+        let toDetail eventSessions (event: Dtos.Event) : Models.EventDetail =
+            { Id = event.Id
+              Date = 
+                  match event.Date with
+                  | None -> DateTime.Today
```
Comment (pallenSL) [inline Publish/Program.fs]: Quietly treating this as a valid Event, _today_, that we are publishing on meetup _right now_.
Possibly not the best approach to dodgy data.  Could be quite embarrassing.

File: Publish/Program.fs [hunk 2]
```diff
@@ -19,21 +19,116 @@ module JsonHttpClient =
...
+              Title = session.Title
+              Description = session.Description
+              SpeakerId = speaker.Id
+              SpeakerForename = speaker.Forename
+              SpeakerSurname = speaker.Surname
+              SpeakerBio = speaker.Bio
+              SpeakerImageUri = speaker.ImageUri
+              SpeakerRating = speaker.Rating
+              StartDate = session.Date
+              EndDate = session.Date |> Option.map (fun date -> date.AddHours(1.0)) }
```
Comment (pallenSL) [inline Publish/Program.fs]: I don't think Publish should be making up the EndDate.
Or anything else.

File: Publish/Program.fs (hunk 1 above)
Comment (jebbinSCL) [inline Publish/Program.fs]: I'll remove it. On a side note, this is copied from the gateway. Do you remember why you added this in the gateway?

File: Publish/Program.fs (hunk 2 above)
Comment (jebbinSCL) [inline Publish/Program.fs]: I'll remove it. On a side note, this is copied from the gateway. Do you remember why you added this in the gateway?

File: Publish/Program.fs (hunk 1 above)
Comment (jebbinSCL) [inline Publish/Program.fs]: I think I will just change this to an option in the gateway as well.

Comment (pallenSL) [inline Publish/Program.fs]: If me, I was possibly fighting with f# option and a non-nullable type.

File: Publish/Program.fs (hunk 2 above)
Comment (pallenSL) [inline Publish/Program.fs]: The Date.addHours() one doesn't sound like me.  No idea.

**Comment by pallenSL** [comment]
:shipit:

File: Publish/Program.fs (hunk 2 above)
Comment (jebbinSCL) [inline Publish/Program.fs]: Actually it wasn't you, it looks like it was much earlier in the history, to much effort to go back and find it exactly.

**Pull request merged**